server/app/models/QuestionTag.java

```diff
@@ -16,10 +17,24 @@ public enum QuestionTag {
   // This question should not be exported in the demographic export csv.
   NON_DEMOGRAPHIC,
   // This question is a universal question, shown with a badge in the UI
-  UNIVERSAL;
+  UNIVERSAL,
+  // These are question tags that are associated with PrimaryApplicantInfoTags.
+  // When adding a new one, ensure you update getPrimaryApplicantInfoTags as well.
```

Comment:
nitpick: since there are multiple methods called `getPrimaryApplicantInfoTags`:
```suggestion
  // When adding a new one, ensure you update getPrimaryApplicantInfoTags below as well.
```